wgpu/examples/framework.rs

```diff
@@ -488,6 +488,26 @@ pub fn test<E: Example>(mut params: FrameworkRefTest) {
 
             example.render(&dst_view, &ctx.device, &ctx.queue, &spawner);
 
+            // Handle specific case for bunnymark
+            if (params.image_path == "/examples/bunnymark/screenshot.png") {
```

Comment:
This feels weird, but I don't currently have a better idea, so thank you!

Reply:
Yep I tried doing a const from bunnymark.rs, but some scariness with the way imports/crates work won't allow it :/